cypress/integration/create.spec.ts

```diff
@@ -0,0 +1,50 @@
+import { ProviderSupport } from '../support/provider'
+
+context('/Create', () => {
+  beforeEach(() => {
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    cy.on('window:before:load', (win: any) => {
+      win.ethereum = ProviderSupport()
+    })
+
+    cy.visit('/create')
+    cy.connectWallet()
+  })
+
+  it('creates a project with all default settings successfully', () => {
+    // Project details
+    cy.findByRole('textbox', { name: /project name/i, timeout: 10000 }).type(
```

Comment:
**suggestion (non-blocking)** this is prolly fine for now, but  I think we should instead opt for a data attribute approach going forward.

Instead of selecting elements based on text - which can change frequently and break tests - we instead add something like `data-testid="next-button"` to elements, and use this to find elements in tests.

Reply:
yep totally agreed! Let's sit down and discuss a format that scales